eclair-core/src/main/scala/fr/acinq/eclair/io/PeerConnection.scala

```diff
@@ -357,6 +364,13 @@ class PeerConnection(keyPair: KeyPair, conf: PeerConnection.Conf, switchboard: A
       case Event(ResumeAnnouncements, d: ConnectedData) =>
         log.info(s"resuming processing of network announcements for peer")
         stay() using d.copy(behavior = d.behavior.copy(fundingTxAlreadySpentCount = 0, ignoreNetworkAnnouncement = false))
+
+      case Event(KillIdle, d: ConnectedData) =>
+        if (!d.isPersistent) {
+          stop(FSM.Normal)
```

Comment:
We should log something, every other place where we stop ourselves, we have a log line to explain why, it will help troubleshooting user issues.